spec/features/cart/user_can_add_items_to_cart_spec.rb

```diff
@@ -25,4 +25,19 @@
       expect(page).to have_content("Total: $10.99")
     end
   end
+
+  it "from show page" do
+    category = Category.create!(name: "potions")
+    item = Item.create!(title: "Love Potion",
+                        description: "make him love you",
+                        price: 10.99,
+                        category: category)
+    visit item_path(item)
+    click_on "Add to Cart"
+
+    within(".notice") do
+      expect(page).to have_content("Love Potion added to cart")
```

Comment:
Test looks great. Glad it's simple and to the point. we don't need to over-complicate these feature test (somewhat talking to myself here)